frontend/src/components/place/PlaceSidebarInfo.vue

```diff
@@ -2,10 +2,20 @@
     <aside class="column is-one-third">
         <div class="place-sidebar">
             <div class="map-box">
-                <img 
-                    src="https://image.ibb.co/isC2MU/Screenshot_from_2018_08_09_00_25_38.png" 
-                    alt="mapbox"
-                >
+                <mapbox
```

Comment:
better to create separate component for that. create and move all map related logic there.
```
<PlaceMapMarker :place="place"  />
```